python/tempo/interpol.py

```diff
@@ -0,0 +1,371 @@
+import sys
+from typing import List
+
+from pyspark.sql.dataframe import DataFrame
+from pyspark.sql.functions import col, expr, first, last, lead, lit, when
+from pyspark.sql.window import Window
+
+# Interpolation fill options
+method_options = ["zero", "null", "back", "forward", "linear"]
```

Comment:
I'd like to suggest we use 'bfill', 'ffill' for the back/forward options.

Reply:
Changed in this commit!

https://github.com/guanjieshen/tempo/commit/015879b1a14b6d37797c9a195bd3f86694c98790